### PR TITLE
Handle touch to update ticks on game board

### PR DESCRIPTION
**Pull request by congtung10t2**

### CODE REVIEW

File: app/src/main/java/com/framgia/carogame/gameView.java [hunk 1]
```diff
@@ -30,16 +51,61 @@ float getDeltaY() {
...
+        float touchY = event.getY();
+
+        int indexX = getIndexX(touchX);
+        int indexY = getIndexY(touchY);
+        if(indexX < 0 || indexX > MAX_WIDTH) return false;
+        if(indexY < 0 || indexY > MAX_HEIGHT) return false;
+        ticks[indexX][indexY] = currentTick;
+        return true;
+    }
+    void UpdateTick() {
```
Comment (vtabk2) [inline app/src/main/java/com/framgia/carogame/gameView.java]: updateTick()

File: app/src/main/java/com/framgia/carogame/gameView.java [hunk 2]
```diff
@@ -30,16 +51,61 @@ float getDeltaY() {
...
+        for(int i = 0; i < MAX_WIDTH; i++) {
+            for (int j = 0; j < MAX_HEIGHT; j++) {
+                if (ticks[i][j] == TickType.SQUARED) {
+                    float side = 20.0f;
```
Comment (vtabk2) [inline app/src/main/java/com/framgia/carogame/gameView.java]: hardcode

**Comment by congtung10t2** [comment]
I just fixed. Please reviews for me. Thanks

**Comment by vtabk2** [comment]
OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**